libs/remote_storage/src/local_fs.rs

```diff
@@ -357,19 +357,28 @@ impl RemoteStorage for LocalFs {
                 .list_recursive(prefix)
                 .await
                 .map_err(DownloadError::Other)?;
-            let keys = keys
+            let objects = keys
                 .into_iter()
-                .filter(|k| {
+                .filter_map(|k| {
                     let path = k.with_base(&self.storage_root);
-                    !path.is_dir()
+                    if path.is_dir() {
+                        None
+                    } else {
+                        Some(ListingObject {
+                            key: k.clone(),
+                            // LocalFs is just for testing, we do not store modification times
```

Comment:
```suggestion
                            // LocalFs is just for testing, so just specify a dummy time
```

File systems usually do store the `mtime` (with all the issues this entails), and we could obtain it if we wanted to, but I agree that it's not worth the coding time to do it.

Also, maybe `SystemTime::UNIX_EPOCH` would be better here, because then it's more obvious that it's just a dummy value, should anyone ever debug code that relies on it.